components/Import/ImportModal.js

```diff
@@ -107,24 +108,22 @@ class Import extends React.Component {
         });
     }
     unmountTrap(){
-
-        if(!this.state.cancelled && !this.state.accepted){ //user clicks outside to cancel without pressing upload button
-            this.setState({
-                cancelled:true
-            });
-
-            if(this.props.ImportStore.fileReadyForUpload){
-                this.context.executeAction(importCanceled, {});
+        if(!this.state.onShowError){
+            if(!this.state.cancelled && !this.state.accepted){ //user clicks outside to cancel without pressing upload button
```

Comment:
For me, when clicking outside the import modal, it doesn't close it so I'm not able to check if this part works